huxley/www/js/stores/CommitteeStore.js

```diff
@@ -5,38 +5,42 @@
 
 'use strict';
 
-var $ = require('jquery');
+var ActionConstants = require('constants/ActionConstants');
+var CommitteeActions = require('actions/CommitteeActions');
 var Dispatcher = require('dispatcher/Dispatcher');
 var ServerAPI = require('lib/ServerAPI');
 var {Store} = require('flux/utils');
 
 
-var _committeePromise = null;
+var _committees = [];
 
 class CommitteeStore extends Store {
-  getCommittees(callback) {
-    if (!_committeePromise) {
-      _committeePromise = ServerAPI.getCommittees();
+  getCommittees() {
+    if (_committees.length) {
+      return _committees;
     }
-    if (callback) {
-      _committeePromise.then(callback);
-    }
-    return _committeePromise;
-  }
 
-  getSpecialCommittees(callback) {
-    var p = this.getCommittees().then((committees) => {
-      return committees.filter((committee) => committee.special);
+    ServerAPI.getCommittees().then(value => {
+      _committees = value;
+      CommitteeActions.committeesFetched();
```

Comment:
This action should contain the `value`, and the Store should only set `_committees` from the data passed into the action.